### PR TITLE
PHPC-1227: Add test for disallowed per-op read/write concerns in transactions

### PR DESCRIPTION
**Pull request by derickr**
This is just a test - not sure which more to add. Documentation still to follow.

### CODE REVIEW

File: tests/session/transaction-integration_error-001.phpt [hunk 1]
```diff
@@ -0,0 +1,70 @@
+--TEST--
+MongoDB\Driver\Session: Setting per-op readConcern or writeConcern in transaction
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_not_replica_set(); ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+<?php skip_if_not_clean(DATABASE_NAME, COLLECTION_NAME); ?>
```
Comment (jmikola) [inline tests/session/transaction-integration_error-001.phpt]: `skip_if_not_clean()`. No need to use arguments unless you're customizing the defaults (e.g. custom collection).

File: tests/session/transaction-integration_error-001.phpt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            'writeConcern' => new \MongoDB\Driver\WriteConcern(
+                \MongoDB\Driver\WriteConcern::MAJORITY ) 
+        ]
+    );
+}, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
```
Comment (jmikola) [inline tests/session/transaction-integration_error-001.phpt]: `--EXPECT--`

File: tests/session/transaction-integration_error-001.phpt (hunk 1 above)
Comment (derickr) [reply tests/session/transaction-integration_error-001.phpt]: Fixed.

File: tests/session/transaction-integration_error-001.phpt (hunk 2 above)
Comment (derickr) [reply tests/session/transaction-integration_error-001.phpt]: I think we should be okay with ``EXPECTF`` — it doesn't make any difference, and makes things easier if tests need to be modified for changes in version — there are plenty of situations where we had to do that.

I'll fix it for this one.

**Pull request merged**